doc/whats_new/v0.23.rst

```diff
@@ -136,6 +136,12 @@ Changelog
   samples in the training set. :pr:`14516` by :user:`Johann Faouzi
   <johannfaouzi>`.
 
+- |Fix| Fixed a bug in :class:`ensemble._bagging.BaseBagging` method
```

Comment:
I think we should document about the public class instead of the base class. It will be more informative for users.

```python
- |Fix| Fixed a bug in :class:`ensemble.BaggingClassifier`
  and :class:`ensemble.BaggingRegressor` where `_get_estimators_indices`
  did not generate the proper indices used during `fit`.
  :pr:`16437` by :user:`Jin-Hwan CHO <chofchof>`.
```

Reply:
Wasn't the original issue related to Isolation Forest? This description is still referencing private API too much. The original issue pertained to a broken estimator_samples_. Was other public functionality broken before this fix?

Reply:
> Wasn't the original issue related to Isolation Forest? This description is still referencing private API too much. The original issue pertained to a broken estimator_samples_. Was other public functionality broken before this fix?

Original issue was related to `IsolationForest` which is a subclass of `BaseBagging` with `ExtraTreeRegressor` as an estimator.

After digging this issue, I found that the bug is in the class `BaseBagging` method `_get_estimators_indices`, and the method is used only in the `estimators_samples_` property.

So all of three classes `BaggingClassifier`, `BaggingRegressor`, and `IsolationForest` have the broken property `estimators_samples_`.

How about the following changes?
```
- |Fix| Fixed a bug in :class:`ensemble.BaggingClassifier`,
  :class:`ensemble.BaggingRegressor`, and :class:`ensemble.IsolationForest`
  where the property `estimators_samples_` did not generate the proper
  indices used during `fit`.
  :pr:`16437` by :user:`Jin-Hwan CHO <chofchof>`.
```

Reply:
Much more relevant to users, thanks